developer-docs-site/docs/guides/sign-a-transaction.md

```diff
@@ -0,0 +1,311 @@
+---
+title: "Signing a transaction"
+slug: "creating-a-signed-transaction"
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating a signed transaction
+
+All transactions exceuted on Aptos blockchain need to be signed, which is a requirement enforced by the chain for security reasons. Before submiting transactions, clients need to hash transactions into bytes and sign the bytes with the account private key (see [account basics][account] for how account and private key works). This guide will walk you through how to create signed transactions.
+
+:::info
+Code examples below are provided in typescript.
+:::
+
+## Overview
+
+Creating signed transactions can be splited into four steps: **preparing the raw transaction (unsigned)**, **getting signing message by serializing the raw transaction**, **signing the serialized transaction to get a signature**, and **deriving a signed transaction from raw transaction and signature**.
+
+Raw transactions are the transactions that have not been signed yet. Raw transactions contain the information about what needs to be executed (transaction payload) and the maximum gas this transaction can consume. A signing message is contructed with a prefix and a body. The body is the serialized bytes of raw transactions by applying the Binary Canonical Serialization ([BCS][bcs]). The signing step is a standard [Ed25519][ed25519] encryption of the signing message with the private key provided by clients. Signing generates a signature that represents the authorization by the client for the chain to run the transaction. Finally, clients need to compose a signed transaction with the original raw transaction and the signature.
+## Key Concepts
+
+### Raw transaction
+Raw transactions we described in this guide means the data input we would submit to the Aptos blockchain for execution. In details, raw transactions are consisted of following data fields:
+* **sender** hex-coded account address of the sender of the transaction
+* **sequence_number** Sequence number of this transaction. This must match the sequence number stored in the sender's account at the time the transaction executes.
+*  **payload** The actual work that the Aptos blockchain needs to do.
+*  **max_gas_amount** Maximal total gas to spend for this transaction.
+*  **gas_unit_price** Price to be paid per gas unit.
+*  **expiration_timestamp_secs** If the current blockchain timestamp is greater than or equal to this time, then the transaction has expired and will be discarded.
```

Comment:
The blockchain timestamp at which the blockchain would either discard this transaction

Reply:
👍